docs/api-docs/users/Account-level-API-accounts.mdx

```diff
@@ -0,0 +1,25 @@
+# Account-level API accounts
+
+## Introduction
+Account-level API tokens enable store owners to manage account-level permissions per BigCommerce account. When generated, this newly created token will belong to the account that store owner has logged into. 
```

Comment:
it is called Account API tokens (based on discussion with Philip Christian and the UX team)